src/RealtimeClient.ts

```diff
@@ -255,6 +256,12 @@ export default class RealtimeClient {
       // remove open handles
       this.heartbeatTimer && clearInterval(this.heartbeatTimer)
       this.reconnectTimer.reset()
+      this.flushSendBuffer()
```

Comment:
thinking if this is the right thing to do, shouldn't just clear `this.sendBuffer` when disconnecting? Do we want to send them even when disconnecting?